### PR TITLE
Docs improvements sources

### PR DESCRIPTION
**Pull request by diegoaguilar**
I think setup with secret API key should be separated in docs.

Also I've added those customer's sources related methods I didn't find in your README first.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -18,12 +18,19 @@ These stripe-node bindings are intended to deal with secret keys, and to take ac
 
 Documentation is available at https://stripe.com/docs/api/node.
 
+## Setup
```
Comment (brandur-stripe) [inline README.md]: Can we call this "Configuration" instead?

File: README.md [hunk 2]
```diff
@@ -18,12 +18,19 @@ These stripe-node bindings are intended to deal with secret keys, and to take ac
...
+
+Make sure you have access to your Stripe API  key, which can be accessed from admin dashboard.
```
Comment (brandur-stripe) [inline README.md]: And can we change this copy to:

``` md
Set your secret API key. This is available in [your dashboard](https://dashboard.stripe.com/account/apikeys).
```

**Comment by diegoaguilar** [comment]
@brandur I did a rebase for last change in configuration instructions. Let me know if you want me to rebase the two methods inclusion commits into one too.

**Comment by diegoaguilar** [comment]
Also, if you can recognize any missing methods I could add them too.

**Comment by brandur-stripe** [comment]
> Also, if you can recognize any missing methods I could add them too.

Thanks! There almost certainly are some, but I don't have an easy way of generating a list. This is great for now though.

**Pull request merged**